src/main/java/racingcar/domain/CarDTO.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.domain;
+
+public class CarDTO {
```

Comment:
```suggestion
public class CarDto {
```
현재는 괜찮지만 DTO와 같은 약자가 반복되면 읽기 힘든 클래스명이 탄생할 수 있습니다 ex)HTTPURLDTO

Reply:
그렇군요...아직 DataTransferObject를 제대로 활용해본 경험이 없어서 깊게 생각하지 않았던 것 같습니다. CarsDTO까지도 같이 수정하겠습니다.!